scss/statics/top/unlogined.static.scss

```
@@ -23,17 +23,19 @@ html{
 
 body {
     $tw-bg-opacity: 1;
-    background-color: hsl(var(--background-default) / $tw-bg-opacity) !important;
     $tw-text-opacity: 1;
-    color: hsl(var(--foreground-default) / $tw-text-opacity);
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
+
+    background-color: hsl(var(--background-default) / $tw-bg-opacity) !important;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)